### PR TITLE
i18n(zh-cn):Create dev-toolbar-app-reference.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/zh-cn/reference/dev-toolbar-app-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,441 @@
+---
+title: 开发者工具栏应用 API
+i18nReady: true
+---
+
+Astro 开发者工具栏应用 API 允许你创建可以用来扩展 Astro 开发者工具栏的应用。这使您可以添加新功能并与第三方服务集成。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/dev-toolbar-app-reference.mdx:6]:
```suggestion
Astro 开发者工具栏应用 API 允许你创建可以用来扩展 Astro 开发者工具栏的应用。这使你可以添加新功能并与第三方服务集成。
```

File: src/content/docs/zh-cn/reference/dev-toolbar-app-reference.mdx [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+
+## 添加应用
+
+[Astro 集成](/zh-cn/reference/configuration-reference/#集成) 可以在 [the `astro:config:setup` hook](/zh-cn/reference/integrations-reference/#astroconfigsetup) 中使用 `addDevToolbarApp` 方法添加应用。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/dev-toolbar-app-reference.mdx:10]:
```suggestion
[Astro 集成](/zh-cn/reference/configuration-reference/#集成) 可以在 [`astro:config:setup` 钩子](/zh-cn/reference/integrations-reference/#astroconfigsetup) 中使用 `addDevToolbarApp` 方法添加应用。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM. Thank you.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**